ownCloud/Static Login/Interface/StaticLoginSingleAccountServerListViewController.swift

```diff
@@ -259,12 +262,43 @@ class StaticLoginSingleAccountServerListViewController: ServerListTableViewContr
 
 	override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
 		if SingleAccountSection(rawValue: section) == .accessFiles {
-			if headerView == nil, let bookmark : OCBookmark = OCBookmarkManager.shared.bookmarks.first, let displayName = self.displayName ?? bookmark.userName {
-				let headerText = String(format: "You are connected as\n%@".localized, displayName)
+			if let bookmark : OCBookmark = OCBookmarkManager.shared.bookmarks.first, let displayName = self.displayName ?? bookmark.userName {
+
+				var avatarView : ResourceViewHost?
+				let avatarSize = CGSize(width: 128, height: 128)
+				let fallbackView = UIImageView(image: UIImage(named: "branding-login-logo"))
+
+				if let avatar = bookmark.avatar {
+					avatarView = ResourceViewHost(viewProvider: avatar, fallbackSize: avatarSize, fallbackView: fallbackView, viewProviderContext: nil)
+				} else {
+					avatarView = ResourceViewHost(fallbackView: fallbackView, viewProviderContext: nil)
+				}
+
+				if let avatarView = avatarView {
+					avatarView.translatesAutoresizingMaskIntoConstraints = false
+
+					NSLayoutConstraint.activate([
+						avatarView.widthAnchor.constraint(equalToConstant: avatarSize.width),
+						avatarView.heightAnchor.constraint(equalToConstant:avatarSize.height)
+					])
+				}
+
+				let attributedTitle = NSMutableAttributedString()
+
+				attributedTitle.append(NSAttributedString(string: displayName.appendingFormat("\n"), attributes: [
+					NSAttributedString.Key.font : UIFont.boldSystemFont(ofSize: 24)
+				]))
+
+				if let serverName = bookmark.url?.host {
+					attributedTitle.append(NSAttributedString(string: serverName, attributes: [
+						NSAttributedString.Key.font : UIFont.systemFont(ofSize: 18)
+					]))
```

Comment:
Font size could be set to `. headline ` (or similar) instead of using float value

Reply:
Thanks. Addressed this in 3036f54cd371fdee1642a7570e1c32354e4d6331 in `feature/graph-api`.